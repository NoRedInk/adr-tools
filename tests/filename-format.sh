ADR_FILENAME_FORMAT="%d-%s" adr new First Record
ls doc/adr
