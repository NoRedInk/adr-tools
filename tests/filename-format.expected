ADR_FILENAME_FORMAT="%d-%s" adr new First Record
doc/adr/1-first-record.md
ls doc/adr
1-first-record.md
